topics/encryption.md

```diff
@@ -119,7 +119,3 @@ versions, ciphers and cipher suites, etc. Please consult the self documented
 ### Performance considerations
 
 TLS adds a layer to the communication stack with overheads due to writing/reading to/from an SSL connection, encryption/decryption and integrity checks. Consequently, using TLS results in a decrease of the achievable throughput per Valkey instance.
-
-### Limitations
-
-I/O threading is currently not supported with TLS.
```

Comment:
Good that this is removed. It could be worth mentioning that it IS supported since 8.0 too, since users may remember that it wasn't supported in the past.

In another PR we can add some dedicated page for IO threading?